Source/WebCore/rendering/RenderListBox.cpp

```diff
@@ -805,13 +805,13 @@ static void setupWheelEventTestMonitor(RenderListBox& renderer)
 
 void RenderListBox::setScrollTop(int newTop, const ScrollPositionChangeOptions&)
 {
```

Comment:
Do you have a bug on file for making RenderListBox::setScrollLeft do something sensible for vertical writing modes?

Reply:
That's https://bugs.webkit.org/show_bug.cgi?id=248262